evaluations/electricity/DeepAREstimator.json

```diff
@@ -1,7 +1,7 @@
 {
-    "ND": 0.10019010231593214,
-    "RMSE": 14238.631410857062,
+    "ND": 0.061101533472537994,
+    "RMSE": 1177.8084716796875,
```

Comment:
This one is interesting: is it because we updated the default number of epochs/batches in the trainer?

Reply:
Probably, you used defaults right?

Reply:
Yes